cmake/templates/ewasm_polyfill.in

```diff
@@ -0,0 +1,13 @@
+// The generation of this file is defined in libyul/CMakeLists.txt.
+// This file was generated by using the content of libyul/backends/wasm/polyfill/@EWASM_POLYFILL_NAME@.yul.
+
+#pragma once
+
+namespace solidity::yul::wasm::polyfill
+{
+
+static char const @EWASM_POLYFILL_NAME@[] = {
+	@EWASM_POLYFILL_CONTENT@, 0
```

Comment:
Wasn't the idea also to split it up so that it works on windows?

Reply:
I thought it will just work like this also on windows. I used the same technique as used for the license. 
`solidity/CMakeLists.txt` states that `Converting to char array is required due to MSVC's string size limit.`

Reply:
ah ok!